src/traces/heatmap/attributes.js

```diff
@@ -79,6 +79,17 @@ module.exports = extendFlat({
             'Picks a smoothing algorithm use to smooth `z` data.'
         ].join(' ')
     },
+    hovergaps: {
+        valType: 'boolean',
+        dflt: true,
+        role: 'style',
+        editType: 'none',
+        description: [
+            'Determines whether or not gaps',
+            '(i.e. {nan} or missing values)',
+            'in the `z` data are hovered on.'
```

Comment:
Maybe

```
  'Determines whether or not gaps',
   '(i.e. {nan} or missing values)',
   'in the `z` data have hover labels associated with them.'
```

would be better. I don't think _gaps in the `z` data are hovered on_ is the proper formulation.

Reply:
Done in https://github.com/plotly/plotly.js/pull/4291/commits/fbaf63656983db2759345038a0365147ad702b06.